.github/workflows/release.yml

```diff
@@ -42,12 +41,51 @@ jobs:
           stripGeneratorNotice: "true"
           issuesWoLabels: "true"
           stripHeaders: "true"
+
       - name: Release fuseml-core
         uses: softprops/action-gh-release@v1
-        if: steps.branch.outputs.BRANCH_NAME == 'main'
         with:
           files: ./bin/*
           body_path: ./CHANGELOG.md
           prerelease: "true"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  docker-iamge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
+        with:
+         version: '3.15.7'
+
+      - name: Install dependencies
+        run: |
+          make deps
+
+      - name: Set image name and tag
+        run: |
+          echo "IMG=ghcr.io/${GITHUB_REPOSITORY}:${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
```

Comment:
Are you adding latest tag too? That's what deployment of fuseml-core is looking for...

Reply:
Do we plan to keep using the latest tag? AFAIK it is not a good practice.